### PR TITLE
Fix issue with units in initial auction bids

### PR DESCRIPTION
**Pull request by dorranh**
Fixes a bug in `start_liquidation_auction_if_possible` where the initial bid for a liquidation auction was calculated assuming the liquidation price was in units of `kit/tez` while it actually has units of `tez/kit`. Also added a unit / regression test which catches the error in the previous implementation.

### CODE REVIEW

**Review by gkaracha — APPROVED** [review]
Nice, thank you so much for fixing this! :pray: LGTM, but let's wait for the CI to finish before merging.

**Comment by dorranh** [comment]
No worries, that sounds good :slightly_smiling_face:

**Comment by utdemir** [comment]
Awesome find, thanks!

**Pull request merged**
